docs/_layouts/default.html

```diff
@@ -4,551 +4,494 @@
eN+0Y7OEczryhLvRhKcRCcn8K0aBf0nnw5IL6hUv5HxvlnV17PT80C/6O5c3bDy6a/dWne7uFWeu+5vY5Vixe8EjL1t0vv72/flrdgvqWNdWzNiyf6fehnOG2CteF65wOOICjcwBC7Pc08mgv+DZ1vePTB3ycHFHr1XpyhL1SG3grsYizeBX+flK6yyPdME1tHD2av+iXJ0WBGr5cpn4R+6ND0lbh1U5+aadqJQuH9yZzwzqPQUUM45NK7OQZm1OCwCmpanqcmjjBEsyFF/bOhIm4jAnFoBsqYkSeMjr3oOE4MYYMFIMLZexC2TDWSQDrxDTWESPjmCqVx9fIeQ65AjZ24gSw0oB/xnH4VjVy1adzEJGCrH/HGRJCPiZWI+QzcM8mEtr2QsPiS/tPbN6/4Is/3P+pbDP4Y2Hmxqa6ZwvVF8mj6svkrVm1GzlCqsVbwou6pqHecF0/8+NG6w0ntHuHXyHeeu45oKVa/n3guRM07juL00ojDZSdnKNHfW3/2ahv7aM/eXxl57JlnSsf/8mjh9bX13/96/X168X61Z1LlvxkDf7uXN3Q1NQAP0zO7wMCahCrAS4b96jmCSOtol2V0omcIRerTBWdAc3spEgruESriaVoQMPqYzTxbIwlc634Xq5AHdikNRfPrEDVWPkssXwCa2rKNP/vE84MTCdbusl2tam7s5Ov30M+p/5sj9pL5nGj10lwn1An4YwL5EevX74tdpIyrJQgGNMWPgQLwcX9tzHiYzSg7U6n/VMullIcHinzZEfK9LlsOAuRxoqXhaXe7rWNrXtpwKypDqPXwwJmEfVRFn+Ng+9wFrC+UJvhYGZyWBdLmqlEMFuwZT1GEW25ksphvJZjT3GU1NDiY6XlKQPzjCRN7ZbGg7TxGP+VlDYQ6Z/5713Y/PrmS+qH4qKzcvNHN/X+j24KMptbwJeTbuFhkHLjUMujbv+0gQXO0Rr9+GPp7jjCPa6u4nM5K1D7dA6s1JTIIDNHU05NvRRgFU/Kw9ZdSAPZYEelOGLJxS4S1AH2EV3FhnAibki3iMKdH3/GPLEoHJtYHvEvWrJ2nqXKP7duw/JFa9V+t78oWhauDOQdftpsrX9k4VfnaHovAXoP7ZkFCKmm6mD3U4K2qrz0xJmHhk2ceUjz6ceYbzDGfJgl5w93/eIXXYfP7//6Y43r1jU2rhdu91682Nv7z5de3fL89u9t3r6d9TBtFc6n9UzChCqG/V9B6tTuXjCyD11iRyCMtqZVTJaqAZZI3LuuGwD5jzGAfNDsZ1hNEHgHyVxWlU8jawqxZnSQwRRJ2f0WLhe985SdocITS/rt1Md0IIMX0U3SUwspaaBlnAYco6C3p0RGjCKWXCFu5IJYKo+ZUs5YMs+Hn81DL9+Xh4c+8D9oi6ZeZKXoPqkn1+pgbZp+Oyup4SS5EBgLQz5sLkM6vYLlV2VaPz2472VBKSHtndjcUbikapvQtHegg/wHadvf1va8+h5xiYc2bx34gJ8bDH9cq/vgruXqE+QPqnvNsWNrSCPalIAr/RwNVxHu51r2GQtMta7A+1CWCuVZCOjJkEYlEz4jWkBjyvZYys9kip92UWF7IJwly+mwqPIIoKVyCC2Kv5iaQ7JbSlpDFdS/qNCwkye9YnDrC4vL0VenlWmjo2k6P8rECJwXMQJn/Nq9yvp5s+ctMr+of3nHlr2/6yIHRuIutLrh29sXzXlk3mxvWW5hybq2tvWb+hevozQ3A/B4CfA4jpvAJbgHOJVLTkGaq4jTgRLy+LjG7EkLapIcdoxVaqnKqRUYB6mKpyoZ3cVjKbEcr2mSgoAbhh6iN4a1fdWUMjHEj1PZQow7ce4Edg4hMrXREyguQ3bsOJdrYqlqRo2TYsnqKfh2dRyocUo1Hk6pBLQ/iEMqCHZl5xeXV6DMKZTkCTXKlJDk6BkfiU6ltDkV7Bh5Im2ufKCGjrEAGRWK4iAuxQni4LMPsUjnpEz3EfOM3Xz9goVf/OtFCx+ZWzf/4Xn1Ow93bNnz1c8vXX14546fkifvI3Fhh2Ve3ZyHEg+umlk9bVbdI5amLc0r5zSazI2LVrduGThzP9mDnAmoUeGI6KW9wO1cspR6TODOFYENMtQJrPVu6VnCWzbZe+ymsDWSCrJm4GA03RZsYoaK38bagoNoqFgFGm7qEXOdRYggkx4QZKHtweA2YVKVM+SxYkDvZGwEnkbidJAUrZihSAt7NUma3RgcOPV4tKx76rTG+XO2tGzcnfua+e7ly7fI5xdMpbHILywYoye46sSvLogvajFK5H3aL6q/wRmA+533d4xahjpGXVHkVk4hFlrdmNUxGvfiy8iu0RdP93b+aGTnqP7G9YFZWvMoN/z+ttHubxzl/mN1rAqgDEfrWq189dTlkZ2rZMv16yPub+U899/fNnT/dOKF2Ki5k3X/BCAgbBgFBdvOXX1T3rTpyfuwoL9+fffuLESk13ET1hGE48+PXEdJeh1InDlWTDD3SDlFxoiSBydBsIjGR5UIXV4JLK+8RpHyNHmYWeZQaH1YiFI/2sobrc1Lo3MqghX5ZZXWDdrJOD+cWO4DxrdidZHf66qs8qxYE8jHg4G5GmSiBtc7AJeTxinXj4TMlYFMispl8VSOyE0AZgrEqMNtvJKyOjg/XLCy+XYl9IxGL61GNDhcNRhg6iF8no9GMR2Kzo3MRbAnPC8b/uyCKA3i7GtZ0K+cU1U1B3+eSAO7ezaczY5Vzc5AvZqdzx6waRT1e+0Csz8o3IatlK5HhTtD2WgNBpjQLxuyBgF2+5VUAVOeLGybtg3TjU6vED7X4nNSoawxhVIWwAgBTYTdxx5jDe0bhWWOtzY2tuJPc200+uCD0WjtSAbiaxqamxvwp2ratKqq6dM5/t57HKe/QXvBHdxyZkXIXHx4E7DVjoYDyEXszbVkenOdaM/LUizTnmtJFwtkGoNZmZ2oA0AdAF6IDLUGw49AgpJL+Cth7+A6/qnBH/A/HjigcudIkHR
```

Comment:
Any reason for the intense indentation here (and for the other lines)?

Reply:
No particular reason, I guess it just lines up perfectly with the `url` above?